documentation/README.md

```diff
@@ -23,27 +23,10 @@ yarn start
 
 This command starts a local, wiki themed development server and opens up a browser window. Most changes are reflected live without having to restart the server.
 
-## Including Markdown or MDX from another file
-*partial.mdx*
+## MDX Features
```

Comment:
Can we have another title for this?

Something broader like Features and tools?

Because we may also explain the cli, the plugin system, default plugins, and so on 🤔